### ISSUE TITLE
superpoint_gauss2

### ISSUE THREAD

**Issue by samadhiiiii** [eric-yyjau/pytorch-superpoint#70]
Thanks for your job.
I have a question about the net,
self.inc = inconv(1, c1)
self.down1 = down(c1, c2)
self.down2 = down(c2, c3)
self.down3 = down(c3, c4)
self.down4 = down(c4, 512)
self.up1 = up(c4+c3, c2)
self.up2 = up(c2+c2, c1)
self.up3 = up(c1+c1, c1)
self.outc = outconv(c1, subpixel_channel)

I cant find what "inconv" ,"down","up" and "outconv" refer to in the code. Is it same to superpoint_pretrained? 
What I mean, is it same as  :
        self.conv1a = torch.nn.Conv2d(1, c1, kernel_size=3, stride=1, padding=1)
        self.conv1b = torch.nn.Conv2d(c1, c1, kernel_size=3, stride=1, padding=1)
        self.conv2a = torch.nn.Conv2d(c1, c2, kernel_size=3, stride=1, padding=1)
        self.conv2b = torch.nn.Conv2d(c2, c2, kernel_size=3, stride=1, padding=1)
        self.conv3a = torch.nn.Conv2d(c2, c3, kernel_size=3, stride=1, padding=1)
        self.conv3b = torch.nn.Conv2d(c3, c3, kernel_size=3, stride=1, padding=1)
        self.conv4a = torch.nn.Conv2d(c3, c4, kernel_size=3, stride=1, padding=1)
        self.conv4b = torch.nn.Conv2d(c4, c4, kernel_size=3, stride=1, padding=1)

**Comment by saunair** [comment]
The definitions are here: https://github.com/eric-yyjau/pytorch-superpoint/blob/4ff74df8fa3c10ce9eb9fdc561f787d9e8bc9691/models/unet_parts.py#L38

They do seem like operations of another architecture, which is the descriptor network. The embedding network seems to be the same

**Comment by EveningLin** [comment]
![image](https://github.com/eric-yyjau/pytorch-superpoint/assets/110521494/92b50d57-704e-42e7-886d-535b37e2bf82)
@saunair @samadhiiiii @guppykang @eric-yyjau @Valeyards I have also noticed that the network results seem to be a bit different. During the down-sampling phase, he also performed a max pooling operation. Could anyone explain the intention behind this? thanks a lot！
我也注意到网络结果似乎有点不同 在down阶段，他还进行了一次maxpool，请问这么做的用意是什么有没有大佬能解释一下呢
